httpserver/external/tomcat-webserver-3.2-license.txt

```diff
@@ -1,58 +0,0 @@
-Name: Apache Tomcat
```

Comment:
Please add a `webserver-3.2.4-license.txt` file. You can start from this file and update version and license information.

Reply:
I don't know why ignore it with /*/external/*/** line in .gitignore. Commit it.